docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "2"
 services:
   couchbase:
-    image: sabrenner/couchbase-server-sandbox:latest
+    image: sabrenner/couchbase-server-sandbox:sandbox
```

Comment:
What's the `sandbox` tag?

Reply:
Just a different version of the image I pushed up. It uses a different parent image, but I wanted to keep the other one in the case that it could still be useful, so I just named this one something different.